src/Azure.Functions.Cli/Helpers/SecurityHelpers.cs

```diff
@@ -109,5 +113,28 @@ internal static X509Certificate2 GetOrCreateCertificate(string certPath, string
 
             throw new CliException("Auto cert generation is currently not working on the .NET Core build.");
         }
+
+        internal static X509Certificate2 CreateCertificateOpenSSL()
+        {
+            const string DEFAULT_PASSWORD = "localcert";
+
+            var cert_file_names = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName().Replace(".", String.Empty));
```

Comment:
nit: for consistency, in C# code we usually use `camelCasing` for variable names. So `certFileName`, etc

Reply:
Ah, Right! Sorry, force of habit. 🤓